x-pack/plugins/cases/server/common/types/case.ts

```diff
@@ -47,6 +47,7 @@ export interface CasePersistedAttributes {
   total_comments: number;
   updated_at: string | null;
   updated_by: User | null;
+  category?: string | null;
```

Comment:
`undefined` for existing cases, `string` if a user sets the category, and `null` if a user deletes the category.